ports/osgearth/portfile.cmake

```
@@ -72,7 +58,8 @@ vcpkg_cmake_configure(
 )
 
 vcpkg_cmake_install()
-vcpkg_cmake_config_fixup(CONFIG_PATH cmake/)
+
+vcpkg_cmake_config_fixup(PACKAGE_NAME osgEarth)
```

```suggestion
vcpkg_cmake_config_fixup()
```
No extra blankline. No explicit PACKAGE_NAME. (It is only used for the directory name, and the default `osgearth` is fine for vcpkg and CMake.)